text/0069-api-v2.md

```diff
@@ -185,7 +192,39 @@ Listed below is the compatibility matrix:
 
 ### Garbage Collection
 
-*To be supplemented in another PR*
+Garbage Collection (`GC` for short) is necessary for recycling the space from:
+
+1. Obsolete versions of [MVCC]
+2. TTL expired entires
+3. (Logically) deleted entries
+
+The **obsolete versions** are those **NOT** the newest of a key, and older than `GC Safe Point`.
+
+`GC Safe Point` is a TSO indicating that versions older than which is safe to be removed. It is advanced by TiDB GC leader, which will keep versions for a life time defaults to 10 minutes. At the same time, services such as [TiDB BR] & [TiCDC] can setup a `Service Safe Point` preventing `GC Safe Point` to advance, and keep more versions to meet service requirement (e.g, [TiDB BR] requires to keep all versions after `backup timestamp` for snapshot consistence). Please refer to [GC Overview] & [complete behavior of TiCDC GC safe point](https://docs.pingcap.com/tidb/dev/ticdc-faq#what-is-the-complete-behavior-of-ticdc-garbage-collection-gc-safepoint) for more details.
+
+There are two garbage collector existing in TiKV. One for transactional data, which utilizes [Compaction Filter] of RocksDB to drop obsolete versions during compaction, and schedules background tasks to physically remove deleted entires. Another one for RawKV data with TTL, which also utilizes [Compaction Filter] to drop TTL expired entries.
+
+`GC` of RawKV is quite similar with transactional data. First of all, check whether it's necessary to run `GC`:
+
+- For each [SST] in a compaction procedure, check [Table Properties] collected by customized collector. A [SST] will be `GC` when:
+  - The [SST] is at the bottommost level (as deleted & expired entries will only be recycled at bottommost level. See below). Or,
+  - There would be obsolete versions (`min_ts` *(minimum timestamp)* of [Table Properties] < `GC Safe Point`. Remind that newest version of a key is **NOT** an obsolete version), and,
+  - More than a ratio (defaults to `10%`) of all versions are obsolete.
```

Comment:
We need to describe situation ```num_rollback_and_locks > props.num_puts as f64 * ratio_threshold``` here?

Reply:
`num_rollback_and_locks` should be zero as RawKV doesn't have rollback or lock records. I think this condition must always be false, so I don't mention it.